spec/rails_helper.rb

```diff
@@ -0,0 +1,34 @@
+require 'simplecov'
+SimpleCov.start
+ENV["RAILS_ENV"] ||= 'test'
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.